docker/test/integration/cluster/ImageStore.py

```diff
@@ -97,10 +105,21 @@ def __build_minifi_cpp_sql_image(self):
 
         return self.__build_image(dockerfile)
 
-    def __build_minifi_cpp_image_with_nifi_python_processors(self, additional_cmd=""):
+    def __build_minifi_cpp_image_with_nifi_python_processors(self, python_option):
         parse_document_url = "https://raw.githubusercontent.com/apache/nifi/rel/nifi-" + NifiContainer.NIFI_VERSION + "/nifi-python-extensions/nifi-text-embeddings-module/src/main/python/ParseDocument.py"
         chunk_document_url = "https://raw.githubusercontent.com/apache/nifi/rel/nifi-" + NifiContainer.NIFI_VERSION + "/nifi-python-extensions/nifi-text-embeddings-module/src/main/python/ChunkDocument.py"
         pip3_install_command = ""
+        requirements_install_command = ""
+        additional_cmd = ""
+        parse_document_sed_cmd = 'sed -i "54d;55d" /opt/minifi/minifi-current/minifi-python/nifi_python_processors/ParseDocument.py && \\'
+        chunk_document_sed_cmd = 'sed -i "112d" /opt/minifi/minifi-current/minifi-python/nifi_python_processors/ChunkDocument.py && \\'
```

Comment:
Good point I also tried doing that first, but the reason changed it is because the dependencies can be listed in multiple lines as it is done in the `ParseDocument.py` at the moment and sed only works in a line-by-line manner.

Reply:
```
sed -i '/class ProcessorDetails:/,/^$/{/^\s*dependencies\s*=/,/\]\s*$/d}' ParseDocument.py
```
`/class ProcessorDetails:/,/^$/`: Do the following between 'class ProcessorDetails:' and the first empty line (so we don't modify other PropertyDescriptor blocks below)
`/^\s*dependencies\s*=/,/\]\s*$/`: Do the following between 'dependencies =' at the start of a line, and ']' at the end of a line
`d`: Delete line

Reply:
Thanks, this seems to work great! We only need to use it inside the alpine container so it should be okay. Updated in 212f3946a3225b38cae974273acf1fb9122d79bd

Reply:
Could you please add a comment with the description of what each part of the pattern means? Regex is write-only, it's not super obvious what is happening there. You could copy it from my previous comment.

Reply:
Sure, added comment in 01357f3906f22c85246c0853f60927d9129c8cb8